docs/src/main/sphinx/security/opa-access-control.md

```diff
@@ -304,6 +308,122 @@ column.
 The same "identity" field may be returned to evaluate column masks under a
 different identity.
 
+### Batch column masking
+
+If column masking is enabled, by default, the plugin will fetch each column
+mask individually from OPA. When working with very wide tables this
+can result in a performance degradation.
+
+Configuring `opa.policy.batch-column-masks` allows Trino to fetch the masks
+for multiple columns in one requests. The requested column are listed under
+the `action.filterResources` node.
+
+An OPA policy supporting batch column masking must return a list containing
+objects with two keys - `viewExpression`, the column mask expression and 
+optional identity, and `index`, the index of the column in the request for
+which the mask applies.
```

Comment:
```suggestion
An OPA policy supporting batch column masking must return a list of objects,
each containing:
- `viewExpression`: an object made of:
    - `expression`: the expression to apply to the column, as a string
    - `identity` (optional): the identity to evaluate the expression as, as a string
- `index`: references the index of the column in the request to which this mask applies
```

Reply:
I wanted to avoid a list but tbf there are other lists in the same doc so it should be fine